src/include/concurrency/transaction_manager.h

```diff
@@ -172,11 +183,14 @@ class TransactionManager {
   // first value is exclusive, last value is inclusive
   std::pair<cid_t, cid_t> dirty_range_ =
       std::make_pair(INVALID_CID, INVALID_CID);
+  
+  bool is_from_transaction_pool_;
```

Comment:
it's temporarily added. I maintain this so that we can still use the older code for running JDBC transactions and use the new code for running microbenchmarks.